blocks/colors/utils.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { find, kebabCase } from 'lodash';
+
+export const getColorValue = ( colors, namedColor, customColor ) => {
```

Comment:
Should we add JSDocs to these functions?

Reply:
These ones are not exported outside of @blocks, that's why it was documented as it is "private", the exported one has the documentation. But I think we can also document it, I will add some docs.